android/sdk/src/main/java/com/taobao/weex/bridge/WXBridgeManager.java

```diff
@@ -453,6 +453,10 @@ public void restart() {
     mWXBridge.resetWXBridge(WXEnvironment.sRemoteDebugMode);
   }
 
+  public void recordViewPort(String instanceId, float viewPort) {
+    mWXBridge.recordViewPort(instanceId, viewPort);
```

Comment:
setViewPortWidth call only when page is create, but sometimes we can create a wxSdkInstance but has not render, so that the page is not create, when we call setViewPortWidth is invalid, anymore setViewPortWidth in C++ will be call by RenderManager, wo we can not record view port width on SetViewPortWidth

Reply:
In this case, you make a litte change to render_manager.set_viewport_width to fit this case.

```
  if (page == nullptr){
    viewports_.insert(std::pair<std::string, float>(page_id, viewport_width));
  }
  else {
    page->set_viewport_width(viewport_width);
  }
```